auto_nag/scripts/workflow/no_severity.py

```diff
@@ -96,12 +95,39 @@ def set_people_to_nag(self, bug, buginfo):
     def get_bz_params(self, date):
         fields = ["triage_owner", "flags"]
         params = {
-            "bug_type": "defect",
             "include_fields": fields,
+            "email1": "intermittent-bug-filer@mozilla.bugs",
+            "emailreporter1": 1,
+            "emailtype1": "notequals",
+            "email2": "wptsync@mozilla.bugs",
+            "emailreporter2": 1,
+            "emailtype2": "notequals",
             "resolution": "---",
-            "f1": "priority",
+            "f1": "component",
```

Comment:
What's the goal to get "component == untriaged" bugs here ? It isn't related to "no_severity" stuff.

Reply:
It's part of the new definition of untriaged.